src/backend/distributed/commands/alter_table.c

```diff
@@ -1589,7 +1581,7 @@ ReplaceTable(Oid sourceId, Oid targetId, List *justBeforeDropCommands,
 		ExecuteQueryViaSPI(query->data, SPI_OK_INSERT);
 	}
 
-	List *ownedSequences = getOwnedSequences(sourceId);
+	List *ownedSequences = getOwnedSequences_internal(sourceId, 0, DEPENDENCY_AUTO);
```

Comment:
To clarify here:
`DEPENDENCY_AUTO` stands for sequences owned as serial
and
`DEPENDENCY_INTERNAL` stands for sequences owned as identity, right?

Here we want only the serial owned sequences, right?
It would be good to add a comment here encapsulating the above.